README.md

```diff
@@ -1,7 +1,151 @@
-# Run Jupyter Notebook action for neuro-flow to be used with Neu.ro template
+# Run a Jupyter Notebook instance
 
-This is a [`neuro-flow`](https://github.com/neuro-inc/neuro-flow) action launching [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/). It's intended to be used with Neu.ro [platform template](https://github.com/neuro-inc/cookiecutter-neuro-project), but can be adapted for other use cases as well.
+This is a [`neuro-flow`](https://github.com/neuro-inc/neuro-flow) action launching a [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/). It's intended to be used with the Neu.ro [platform template](https://github.com/neuro-inc/cookiecutter-neuro-project), but can be adapted for other use cases as well.
 
-It requires image name to run Notebook on as well as references to 5 volumes: data, code, config, notebooks and results, which will be mounted to `/project/data`, `/project/modules`, `/project/config`, `/project/notebooks` and `/project/results` respectively.
+It requires the name of the image on which to run Notebook, as well as references to 5 volumes: data, code, config, notebooks and results. These volumes will be mounted to `/project/data`, `/project/modules`, `/project/config`, `/project/notebooks`, and `/project/results` respectively.
 
-After launch web UI open up in default browser automatically.
\ No newline at end of file
+After the Notebook instance is launched, its Web UI will be automatically opened in the default browser.
+
+### Quick example:
+
+```
+jobs:
+  jupyter:
+    action: gh:neuro-actions/jupyter@master
+    args:
+      image: neuromation/base
```

Comment:
Let's set `neuromation/base:latest` as a default image used by this action (and update the action description respectively).
WDYT?